### DISCUSSION TITLE
Team A - Course 3

### DISCUSSION THREAD

**Discussion by VSFe** [lablup/contribution-academy-2021#71] [Show and tell]
1. `backend.ai run -c 'print("Hello")' python` 실행 결과
![course3_1](https://user-images.githubusercontent.com/4595546/130355461-e48ea52e-6900-4a47-a27f-dc985b446c9e.PNG)

2. fashion mnist 실행 결과 
![course3_2](https://user-images.githubusercontent.com/4595546/130355464-93f920bb-e86c-43e4-8d78-250e674457f6.PNG)

**Comment by eunjungchoi** [comment]
A팀: 최은정

1. `backend.ai run -c 'print("hello")' python` 실행
<img width="1179" alt="스크린샷 2021-08-22 오전 3 05 59" src="https://user-images.githubusercontent.com/5889435/130331127-1f84eade-153a-41c6-85a3-9b3577351297.png">

2. `backend.ai run --rm --exec 'python single_mnist.py' tf2 ./single_mnist.py ./mnist.py` 실행 
<img width="1225" alt="스크린샷 2021-08-22 오후 9 32 41" src="https://user-images.githubusercontent.com/5889435/130355352-b29101b6-efb1-4b81-8f11-2a9872fae3a4.png">
